src/routes/docs/products/functions/execute/+page.markdoc

```diff
@@ -721,11 +733,43 @@ const promise = functions.createExecution(
     );
 
 promise.then(function (response) {
-    console.log(response);
+    console.log(response); // Success
 }, function (error) {
-    console.log(error);
+    console.log(error); // Failure
 });
 ```
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/appwrite/sdk-for-go/appwrite"
+)
+
+func main() {
+	client := appwrite.NewClient(
+		appwrite.WithProject("<PROJECT_ID>"),
+	)
+
+	functions := appwrite.NewFunctions(client)
+
+	response, err := functions.CreateExecution(
+		"<FUNCTION_ID>", // functionId
+		functions.WithCreateExecutionBody("<BODY>"), // body (optional)
+		functions.WithCreateExecutionAsync(true), // Scheduled executions need to be async
+		functions.WithCreateExecutionPath("<PATH>"), // path (optional)
+		functions.WithCreateExecutionMethod("GET"), // method (optional)
+		functions.WithCreateExecutionHeaders(map[string]interface{}{}), // headers (optional)
+		functions.WithCreateExecutionScheduledAt("2025-10-15T06:38:00.000+00:00")) // Schedule execution (optional)
+
+	fmt.Println(response)
```

Comment:
let's use `execution` for the variable name to make it consistent with the other go snippet.